### ISSUE TITLE
instagram not working

### ISSUE THREAD

**Issue by soundreactor** [jabbany/knProxy#13]
instagram is not usable. : (
would it be hard to fix that?

**Comment by jabbany** [comment]
Yeah, instagram uses lots of JS magic to load resources so I'm not surprised it would break. knProxy can only handle very very basic rewriting of JS requests and those using relative URLs will fail quite badly.

Some sites have a non-JS mobile version but I'm not sure instagram does.

Not sure that it would be completely impossible but I would need to look into it some more.
